.nrepl-port

```diff
@@ -0,0 +1 @@
+55072
```

Comment:
Should this file really be added?

What if somebody else wants to use a different port, we shouldn't enforce our choice on them. So maybe this file can be removed from the pull request? 

I'm not demanding this, just brining it up for discussion. Do raise your opinion.